frontend/src/css/Sidebar.styl

```diff
@@ -1,17 +1,20 @@
 sidebar-animate-speed = 0.5s
 sidebarFontColorHover = grey
 borderWhenActive= 5px
+spacingInsideMenu = 40px
```

Comment:
Shouldn't we put theses variables in ``variables.styl``?

Reply:
Since they are specific to this component and will not be reused in other contexts, I put them here because it makes them easier to manipulate, but I have no problem changing them if you think it's better.